webpack.config.js

```diff
@@ -81,12 +79,12 @@ module.exports = config(({ development, production, version }) => ({
       {
         from: 'chrome-manifest.json',
         to: 'chrome/manifest.json',
-        transform: addVersion
+        transform: transformChromeManifest
       },
       {
         from: 'firefox-manifest.json',
         to: 'firefox/manifest.json',
-        transform: addVersion
+        transform: transformFirefoxManifest
```

Comment:
You can probably reuse a significant amount of code by taking an additional argument in this function

```
transformManifest(browser: 'firefox' | 'chrome')
```

Then in the function just conditionally add the chrome specific permissions by checking `browser` variable.